plugins/outputs/prometheus_client/README.md

```diff
@@ -1,6 +1,13 @@
 # Prometheus Client Service Output Plugin
 
-This plugin starts a Prometheus Client, listening on a port defined in the
-configuration file.
+This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all metrics on `/metrics` to be polled by a Prometheus server.
```

Comment:
The original wording made more sense to me when the port was the only configuration option. When the following block shows the example config it seems odd to specifically call out the port here. 

Additionally the config was referred to as port here and address within the example config.